### PR TITLE
Replaced all old migrations with a new one.

### PR DESCRIPTION
**Pull request by hbjorgo**

### CODE REVIEW

File: src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.Designer.cs [hunk 1]
```diff
@@ -609,7 +612,6 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                         .HasColumnType("int");
 
                     b.Property<string>("Description")
```
Comment (bratli) [inline src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.Designer.cs:614]: Er ikke description required lenger?

File: src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs [hunk 2]
```diff
@@ -0,0 +1,1190 @@
...
+                        principalColumn: "Id");
+                    table.ForeignKey(
+                        name: "FK_Projects_Persons_ModifiedById",
+                        column: x => x.ModifiedById,
+                        principalTable: "Persons",
+                        principalColumn: "Id");
+                });
+
+            migrationBuilder.CreateTable(
+                name: "RequirementTypes",
```
Comment (bratli) [inline src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs:156]: Stemmer det at RequirementTypes sin nyeste kolonne, Icon, ikke skal være med her?

File: src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.Designer.cs (hunk 1 above)
Comment (hbjorgo) [reply src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.Designer.cs:614]: Apparently not. It's not in the configuration code, and it's nullable in the db.

File: src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs (hunk 2 above)
Comment (hbjorgo) [reply src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs:156]: That's on another feature branch, not merged into master yet. I see no reason to wait for it to be merged.

Comment (kfaln) [reply src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs:156]: It's in master now... might cause a nasty merge conflict

Comment (hbjorgo) [reply src/Equinor.Procosys.Preservation.Infrastructure/Migrations/20200708132753_InitialDb.cs]: Merged master and replaced the new migration with a new one.

**Pull request merged**